transport/http/server.go

```diff
@@ -226,6 +233,7 @@ type Headerer interface {
 
 type interceptingWriter struct {
 	http.ResponseWriter
+	http.Hijacker
```

Comment:
This isn't safe — if this field is nil, `h, ok := w.(http.Hijacker)` will succeed, but calling `h.Hijack()` will panic, as `h` will be nil.